recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "rubin-env" %}
-{% set version = "9.1.0" %}
+{% set version = "10.0.0" %}
 {% set build_number = 0 %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 build:
-  skip: True  # [python_impl == 'pypy' or win or py != 311]
+  skip: True  # [python_impl == 'pypy' or win or py != 312]
```

Comment:
You might consider moving to 3.13? numba just shipped on 3.13 which was the big blocker.

Reply:
That's great news, and I will defintely suggest we start 11dev on 3.13 to see what happens. Today though I have to stick to 3.12 if there is any chance of getting this out early next week.